src/contexts/createBouquet/datasetProperties.js

```diff
@@ -0,0 +1,146 @@
+/**
```

Comment:
Par exemple ici, on pourrait rajouter une documentation avec des examples :

```js
\**
* Dataset properties give additional context to each dataset in a bouquet.
* 
* Imagine the following `dataset`:
* 
* ```json
* {
*   "id": "1a2s3d4f",
*   "title": "Données thermofluviales",
*   "description": "Description qui ne donne pas indice du contexte d'utilisation de la donnée"
* }
* ```
*
* If we add that dataset to a `bouquet` (`topic`), we'd still lack that context, to know what the data can be 
* used for:
*
* ```json
* {
*   "title": "Indicateurs thermofluviaux",
*   "description": "...",
*   "tags": ["ecolab", "theme:agir"],
*   "datasets" : [
*     {
*       "id": "1a2s3d4f",
*       "title": "Données thermofluviales",
*       "description": "Description qui ne donne pas indice du contexte d'utilisation de la donnée"
*     }
*   ]
* }
* ```
*
* In order to give more context, we use an intermediate entity called {DatasetProperties}. For example, to complete 
* our previous example, imagine that the dataset comes from `data.gouv.fr`:
* 
* ```json
* {
*   "title": "Indicateurs thermofluviaux",
*   "description": "...",
*   "tags": ["ecolab", "theme:agir"],
*   "datasets" : [
*     {
*       "id": "1a2s3d4f",
*       "title": "Données thermofluviales",
*       "description": "Description qui ne donne pas indice du contexte d'utilisation de la donnée"
*     }
*   ],
*   "datasets_properies": [
*     {
*       "name": "The name of the properties (not the dataset)",
*       "description": "Why is this dataset associated with the bouquet",
*       "datagouvId": "1a2s3d4f"
*     }
*   ]
* }
* 
* Etc.
* 
* ```
*/
```

Reply:
poke @YeLnatSs @streino et al.

Reply:
J'ai l'impression que ces exemples peuvent être des tests (et donc du code) plutôt que de la doc, non ? L'avantage c'est que les tests risquent moins de devenir incohérents avec le code que la doc.